packages/tools/src/utilities/getViewportsForAnnotation.ts

```diff
@@ -0,0 +1,31 @@
+import { getEnabledElements, utilities as csUtils } from '@cornerstonejs/core';
+import type { Annotation } from '../types';
+
+const { isEqual } = csUtils;
+
+/**
+ * Finds a all matching viewports in terms of the orientation of the annotation data
+ * and the frame of reference. This doesn't mean the annotation IS being displayed
+ * on these viewports, just that it could be by navigating the slice, and/or pan/zoom,
+ * without changing the orientation.
+ *
+ * @param annotation - Annotation to find the viewports that it could display in
+ * @returns All viewports to display in
+ */
+export default function getViewportsForAnnotation(annotation: Annotation) {
```

Comment:
we have this already 

packages/tools/src/utilities/getViewportForAnnotation.ts

Reply:
ViewportS (plural)
Later it looks for the viewport that is in a toolGroup with freehand tool

Reply:
Pushed a change to make `getViewportForAnnotation` call `getViewportsForAnnotation` removing duplicated code